src/gui/AutomationPatternView.cpp

```diff
@@ -89,6 +96,22 @@ void AutomationPatternView::update()
 
 
 
+
+void AutomationPatternView::allowOpenToBar()
+{
+	if(Engine::getSong()->playMode() == 1)
+	{ 
+		TextFloat::displayMessage("Right Playmode, allow");
```

Comment:
If graying out "open to playhead" based on playMode is ok, this should allow the option to be selected in context menus. Right now it just displays a message.